plugins/woocommerce-beta-tester/includes/class-wc-beta-tester.php

```
@@ -393,7 +516,129 @@ protected static function is_rc_version( $version_str ) {
+		return null;
+	}
+
+	/**
+	 * Callback for upgrader_process_complete action.
+	 * Stores the nightly timestamp when WooCommerce is updated via standard updater.
+	 *
+	 * @since 3.1.0
+	 * @param WP_Upgrader $upgrader WP_Upgrader instance.
+	 * @param array       $options  Array of update data.
+	 */
+	public function on_upgrade_complete( $upgrader, $options ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$upgrader'. (undefined)

(UnusedFormalParameter)